maintenance/monitor/monitor-component-metrics.md

```diff
@@ -120,6 +126,36 @@ spec:
 EOF
 ```
 
+##### **kube-controllers configuration**
+
+kube-controllers prometheus metrics are **enabled** by default on TCP port 9094. You can adjust the port by modifying the KubeControllersConfiguration resource.
+
+```bash
+calicoctl patch kubecontrollersconfiguration default  --patch '{"spec":{"prometheusMetricsPort": 9095}}'
```

Comment:
Should this command use `9094` instead of `9095`?

Reply:
I was meaning to show that you can change the deafult - 9094 would be the same as the default.